plugins/constraints/src/external.constraint.e2e.spec.ts

```
@@ -38,6 +52,16 @@ describe('externalConstraint()', () => {
 
   afterAll(async () => {
     await deleteSpace(spaceId)
+
+    driver.shutdown()
+  })
+
+  beforeEach(() => {
+    listener.resetCount()
+  })
```

Biome

[error] Disallow duplicate setup and teardown hooks.

Disallow beforeEach duplicacy inside the describe function.

(lint/suspicious/noDuplicateTestHooks)